tests/lib/snaps/test-snapd-daemon-notify/meta/snap.yaml

```diff
@@ -3,8 +3,12 @@ version: 1.0
 summary: Basic daemon-notify snap
 description: A basic snap which allow sending notification messages to systemd through the notify socket
 
+hooks:
+  install:
+    plugs: [network]
```

Comment:
Why is the network plug necessary?

Reply:
hmm it might not be actually, I thought it was but it's been awhile since I've looked at this

Reply:
the network plug didn't end up being necessary and it was for the install hook anyways so it wouldn't have affected the main daemon being tested here